frontend/app/src/store/statusdata.js

```diff
@@ -12,7 +12,7 @@ export const useStatusDataStore = defineStore('statusdata', {
   actions: {
     async fetchGetAllStatus(includeDeleted = false) {
       try {
-        const { data } = await statusService.serviceGetAllStatus(
+        const { data } = await rowStatusService.serviceGetAllStatus(
```

Comment:
is the prefix "service" a convention in naming service methods for example serviceGetAllStatus or it should be getAllStatuses(...)

Reply:
For the Dev convenience, we used the function name starting with the Service keyword so that the Dev knows that the function is written in the Service Layer. For reference, we have the same function in the status store called getAllStatuses. So to differentiate the function names we add a service to each function.

Reply:
The differentiation is achieved by appropriately naming the variable. It's not uncommon to have the same names in different types. Another possibility is to use slightly different method names (not just using prefix) if possible. However, just using the proper name for the variable is often sufficient